docs/sce_file_contents.md

```diff
@@ -45,15 +45,16 @@ The following per-cell data columns are included for each cell, calculated using
 | `subsets_mito_percent`  | Percent of all UMI counts assigned to mitochondrial genes                                                                                                                                     |
 | `total`                 | Total UMI count for RNA-seq data and any alternative experiments (i.e., CITE-seq)                                                                                                             |
 
-The following are additional per-cell data columns included only in `filtered` objects.
+The following are additional per-cell data columns included in both the `filtered` and `processed` objects.
 These metrics were calculated by using [`miQC`](https://bioconductor.org/packages/release/bioc/html/miQC.html), a package that jointly models proportion of reads belonging to mitochondrial genes and number of unique genes detected to predict low-quality cells.
+Cells with a high likelihood of being compromised as calculated by `miQC` (greater than 0.75) and cells that do not pass a minimum number of unique genes detected threshold of 200 are removed from the counts matrix found in the `processed` objects.
```

Comment:
I reworded somewhat. I'm not sure we need the detail about the filtering here: I moved the link to the processed gene expression data section up here for brevity. (So line 59 should be deleted)

Secondary thought: we should rename `ccdl_filter` to `scpca_filter`.

```suggestion
These include metrics were calculated by [`miQC`](https://bioconductor.org/packages/release/bioc/html/miQC.html), a package that jointly models proportion of reads belonging to mitochondrial genes and number of unique genes detected to predict low-quality cells.
We also include the filtering results used for the creation of the `processed` objects. 
See the description of the {ref}`processed gene expression data <processing_information:Processed gene expression data>` for more information on filtering performed to obtain the `_processed.rds` file.
```

Reply:
Filed https://github.com/AlexsLemonade/scpca-nf/pull/239 for the secondary thought... We should update all docs here if that is approved.